architecture/core_concepts/routes.adoc

```diff
@@ -245,15 +245,18 @@ $ oc set env dc/router ROUTER_SYSLOG_ADDRESS=127.0.0.1 ROUTER_LOG_LEVEL=debug
 |`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
 |`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named *_tls.crt_*. If *_tls.crt_* is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if `DEFAULT_CERTIFICATE` or `DEFAULT_CERTIFICATE_PATH` are not specified.
 |`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if `DEFAULT_CERTIFICATE` is not specified.
+|`*DROP_SYN_DURING_RESTART*` |  false | Whether to enable an iptables rule to drop certain packets during a restart to provide a seamless restart.  See xref:../../install_config/router/default_haproxy_router.adoc#preventing-connection-failures-during-restarts[the install guide] for details.
```

Comment:
`false`

    When set to `true` enables an iptables rule

s/install guide/Installation Guide